CHANGELOG.md

```diff
@@ -1,16 +1,3 @@
-## [2.12.0] - 2023-03-10
-FEATURES:
-
-- Adds the optional `randomization_units` attribute to the `launchdarkly_metric` resource and data source. For more information, read [Allocating experiment audiences](https://docs.launchdarkly.com/home/creating-experiments/allocation). Thanks, @goldfrapp04!
-
-- Bumps golang.org/x/net dependency from 0.1.0 to 0.7.0.
-
-- Bumps Go version to 1.19.
-
-NOTES:
-
-- Fixes incorrect attribute casing in `launchdarkly_metric` resource and data source documentation.
-
```

Comment:
Ugh this shouldn't be deleted.

Reply:
yeah i'm going to update it manually and push through